### ISSUE TITLE
Hoverstatus

### ISSUE THREAD

**Issue by marcohanke** [FriendsOfREDAXO/cookie_consent#13]
Wenn ich als Button-Hintergrund weiß wähle, ist der hoverstatus per js schwarz. Generell wäre ein Theme schön dass sich ausschließlich um die Ausrichtung und nicht um Schriftgrößen, Borderradius, etc. kümmert.

**Comment by chrison94** [comment]
Das Add-On stellt lediglich das Skript von Insites zur Verfügung und der Konfigurator bietet die Konfigurationsmöglichkeiten die Insites dem Nutzer direkt bietet. Die Borderfarbe etc. wird meiner Meinung nach automatisch berechnet. [https://cookieconsent.insites.com/documentation/javascript-api/](https://cookieconsent.insites.com/documentation/javascript-api/) die Erklärung bei Palette wird vielleicht dein Farb-Problem beheben. Die Font-size kannst du in deinem CSS anpassen, das Add-On bietet ja nur die Grundlage für das Cookie Problemchen. :-)

**Comment by marcohanke** [comment]
Klar, ich dachte an ein extra Theme "clean" oder sowas mit so wenig css und styles wie möglich (übersteigt meine Fähigkeiten). Aber es lief auch mit css rauskopieren und anpassen ;-) Die Farbe passt er über die Kontrastfarbe an. Von weiß ist das halt schwarz, ging aber auch mit überschreiben.

**Comment by bloep** [comment]
@marcohanke Ich würde ein Theme "clean" anlegen, dass keinerlei Styles von seitens des AddOns vorgibt. Dennoch bringt Insites Cookies-Consent selber Styles mit. Aber wie du schon sagtest können diese per CSS einfach überschrieben werden.
So in Ordnung?

**Comment by marcohanke** [comment]
@bloep Ich finde es halt irgendwie ungewöhnlich das Schriften und Formen über ein Addon mitgegeben werden. Erinnert so ein bisschen an die schlechten alten Wordpress Tage ;-) Klar kann ich das überschreiben, mache ich im Moment auch. Ich fände es nur irgendwie sauberer, wenn per CSS nur die Basics mitgegeben werden, also Position unten, Button neben Text und sowas. Der Rest passt dann ja in der Regel automatisch und man muss auch nichts überschreiben. Ist aber so wie es ist auch nicht dramatisch und funktioniert ja bei allen Prima.

**Comment by tbaddade** [comment]
@marcohanke Wenn du die Option „CSS automatisch einbinden“ deaktivierst, müsste dir geholfen sein?

**Comment by bloep** [comment]
@tbaddade ne, diese Option gibt es so gar nicht. siehe [README.md](https://github.com/friendsofredaxo/cookie_consent/blob/master/README.md) unter Verwendung

**Comment by marcohanke** [comment]
Im Moment habe ich mir die CSS kopiert, abgespeckt (gerade was Schriftart, Größe, etc. angeht und sie händisch eingebunden, komme also klar. Wenn du willst kannst du das gerne als Clean mit anlegen, dann wäre es auch update Sicher und ich müsste nicht immer händisch meine CSS einbinden ;-)

**Comment by bloep** [comment]
@marcohanke Wäre auch mein Vorschlag gewesen, dass diese verwendet wird, wenn als Theme "clean" ausgewählt wird

**Comment by marcohanke** [comment]
Top!

**Comment by tbaddade** [comment]
Das hatte ich dann falsch interpretiert hier im Code. Ich fänds gut, wenn man separat für CSS in Js entscheiden könnte. Dass Css könnte so auch direkt im Sass definiert werden.

**Comment by bloep** [comment]
@tbaddade ok, mann sollte dies dann auch direkt für das JS aufnehmen, sodass man genauer entscheiden kann, welche Skripte ausgegeben werden sollen.
